### PR TITLE
Issue #259 run single test/test case/package

### LINKED ISSUE #259: Document how to run single test/test case/package in development section
Currently, in the [development section#running-tests](https://github.com/minishift/minishift/blob/master/docs/developing.md#running-tests), there is no instruction how one can run single test/ test case or package.

It is needed for faster development as running entire tests with `make test` may take time.

Document the possible ways to test single test/test case or package.

### PR DESCRIPTION
**Pull request by budhrg**
Fix #259

### CODE REVIEW

File: docs/developing.md [hunk 1]
```diff
@@ -151,12 +151,31 @@ For more options refer to the _minishift_ [synopsis](./minishift.md).
...
+- Run specific test case
+```
+go test -v <relative path of package> -run <Testcase Name>
+```
+
+- Run tests matching pattern
+```
+go test -v <relative path of package> -run "Test<Regex pattern to match tests>"
+```
+
```
Comment (praveenkumar) [inline docs/developing.md]: I would suggest to add `go test --help` at the end for reference if someone want to know what other flags they can use.

Comment (budhrg) [inline docs/developing.md]: Added @praveenkumar :+1:

Comment (hferentschik) [inline docs/developing.md]: You probably would like to indent this as part of the enum, right? Now your code blocks are outside. I'll take care of this.

Comment (budhrg) [inline docs/developing.md]: ok @hferentschik

**Comment by hferentschik** [comment]
Merged, thanks. I adjusted the code block formatting though.

**Pull request closed without merging**